sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -217,9 +217,60 @@
 #--------------------------------------------------------------------------------------------------
 # LOGGING
 
-# Level of logs. Supported values are INFO(default), DEBUG and TRACE (DEBUG + SQL + ES requests)
+# SonarQube produces logs in 4 logs files located in the same directory (see property sonar.path.logs below),
+# one per JVM: App (main process), Web Server (aka. Web), Compute Engine (aka. CE) and Elastic Search (aka. ES)
```

Comment:
I suggest to replace "JVM" by "process". Moreover correct spelling of ES is "Elasticsearch".